staging/src/k8s.io/apiserver/pkg/util/wsstream/conn_test.go

```diff
@@ -141,7 +142,7 @@ func TestBase64Conn(t *testing.T) {
 		defer wg.Done()
 		data, err := ioutil.ReadAll(conn.channels[0])
 		if err != nil {
-			t.Fatal(err)
+			t.Error(err)
```

Comment:
Return early here?

Reply:
Oh! I think I may have misunderstood @tiloso's suggestion. @tiloso can you please confirm if there should be a return statement here? (In the previous comment I thought that you had asked me to remove return from here)